samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -691,6 +691,19 @@ To test this feature, complete the following steps:
       #. Start the ringing action on your device by tapping the :guilabel:`Play sound` button.
       #. Observe that **LED 1** is lit to confirm that the Android device is able to connect to your development kit after a clock synchronization.
 
+Disablement instructions
+------------------------
+
+To disable the locator tag device, complete the following steps:
+
+.. note::
+   Disablement instructions are used in Android and iOS to provide information on how to disable unknown trackers from each vendor.
+   These disablement instructions section is associated with the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section) and is used for demonstration purposes.
```

Comment:
Is this a best place to include the note? Since above we're referring to listed steps (``complete the following steps:``), I feel those steps should appear right below the statement. Maybe we could rework the paragraph and get rid of the note completely? E.g.,

```suggestion
The following instructions explain how to disable unknown trackers from each vendor on Android and iOS devices.
They apply to the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section), and are used for demonstration purposes.
To disable the locator tag device, complete the following steps:

.. note::
```

Reply:
> The following instructions explain how to disable unknown trackers from each vendor on Android and iOS devices.

This sentence changed the meaning, our disablement instructions are only relevant for our model and the Nordic DK/Thingy:53. The previous sentence was supposed to be informational and explain why mobile platforms (iOS / Android) need disablement instructions.

Reply:
thanks for clarifying - didn't catch that